grades/models.py

```diff
@@ -16,27 +15,46 @@ class Course (models.Model):
         ]
     )
 
+    def __str__(self):
```

Comment:
This file is being updated by you in PR #62, it doesn't look like it is supposed to be here as well.

Reply:
@simzacks I add it in #62 now.